### PR TITLE
Bump to v3.2.0-dev

### PR DESCRIPTION
**Pull request by mheon**
Bump version to reflect that we now have a v3.1 branch.

### CODE REVIEW

**Comment by Luap99** [comment]
Tests are not happy

File: version/version.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var Version = semver.MustParse("3.1.0-dev")
 var APIVersion = map[Tree]map[Level]semver.Version{
 	Libpod: {
 		CurrentAPI: Version,
-		MinimalAPI: semver.MustParse("3.0.0"),
+		MinimalAPI: semver.MustParse("3.2.0-dev"),
```
Comment (Luap99) [inline version/version.go:41]: I don't think we have to bump the minimal API version. @jwhonce PTAL

Comment (mheon) [reply version/version.go:41]: The tests start blowing up if we don't - the Current API version is set to v3.2.0-dev, and according to semantic versioning, v3.2.0 is more recent than v3.2.0-dev (suffixes indicate pre-release). Unless we bump this. the minimum API version is lower than the current API version.

Comment (Luap99) [reply version/version.go:41]: I don't think so, looking at the changes I can see that you changed this from 3.0.0 so it worked together with the 3.1.0-dev version.

Comment (jwhonce) [reply version/version.go:41]: @Luap99 @mheon I would suggest that MimimalAPI be n-1 (Current release version) at least at until the end of the release cycle for a given branch. As we stabilize the API, I could see a day where the drift between Current and Minimal grows.

**Pull request merged**